src/mlpack/tests/ann_layer_test.cpp

```diff
@@ -2678,31 +2688,31 @@ TEST_CASE("AtrousConvolutionLayerPaddingTest", "[ANNLayerTest]")
 TEST_CASE("LayerNormTest", "[ANNLayerTest]")
 {
   arma::mat input, output;
-  input << 5.1 << 3.5 << arma::endr
-        << 4.9 << 3.0 << arma::endr
-        << 4.7 << 3.2 << arma::endr;
 
+  input = { {5.1, 3.5},
+            {4.9, 3.0},
+            {4.7, 3.2} };
   LayerNorm<> model(input.n_rows);
   model.Reset();
 
   model.Forward(input, output);
   arma::mat result;
-  result << 1.2247 << 1.2978 << arma::endr
-         << 0 << -1.1355 << arma::endr
-         << -1.2247 << -0.1622 << arma::endr;
-
+ 
+result = { {1.2247,  1.2978},
+           {0, -1.1355},
+           {-1.2247 , -0.1622} };
   CheckMatrices(output, result, 1e-1);
   result.clear();
 
   output = model.Mean();
-  result << 4.9000 << 3.2333 << arma::endr;
-
+ 
+ result ={ 4.9000,3.233};
```

Comment:
```suggestion
 result ={ 4.9000, 3.233};
```

Reply:
@GauravSarkar No need to rush. Make sure the layout is nice and clean. This helps with maintaining the code.  
Ensure the spacing is consistent, columns line up, etc.  Examples:

    X = { 4.9000, 3.233 };

Note the space after `=`, after `{`, after `,`, and before `}`.

    X = { { 5.1, 3.5, 1.4 },
          { 4.9, 3.0, 1.4 },
          { 4.7, 3.2, 1.3 } };

Note how the columns line up.

Reply:
input = { { 1,  446, 42 },
            { 2,  16,  63 },
            { 3,  13,  63 },
            { 4,  21,  21 },
            { 1,  13,  11 },
            { 32, 45,  42 },
            { 22, 16,  63 },
            { 32, 13,  42 } };
Is this layout correct?

Reply:
If it follows the format Conrad posted above it's correct.